custom_components/opnsense/update.py

```
@@ -31,111 +31,100 @@
+        try:
+            self._attr_installed_version = dict_get(
+                state, "firmware_update_info.product.product_version"
+            )
+        except (TypeError, KeyError):
+            self._attr_installed_version = None
 
         try:
-            # fake a new update
-            # return "foobar"
             product_version = dict_get(
                 state, "firmware_update_info.product.product_version"
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)